modules/serverless-kafka-source-yaml.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_sources/serverless-kafka-source.adoc
+
+[id="serverless-kafka-source-yaml_{context}"]
+= Creating a Kafka event source using YAML
+
+You can create a Kafka event source by using YAML.
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource are installed on your cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+.Procedure
+
+. Create a YAML file containing the following:
++
+[source,yaml]
+----
+apiVersion: sources.knative.dev/v1beta1
+kind: KafkaSource
+metadata:
+  name: <source-name>
+spec:
+  consumerGroup: <group-name>
+  bootstrapServers:
+  - <list-of-bootstrap-servers>
+  topics:
+  - <list-of-topics>
+  sink:
+----
```

Comment:
Added a line for each, but these are more complex Kafka specific concepts, so I've added an additional resources link too for the kafka concepts AMQ streams docs. Just to give context: Kafka is an optional event source, so generally I think we should assume a level of knowledge here, i.e. that someone wanting to use it would know what these things are and how they work, especially since they will require a Kafka cluster etc to use the Kafka bit for serverless.

Reply:
Great!